packages/relay/src/lib/eth.ts

```diff
@@ -1630,7 +1632,7 @@ export class EthImpl implements Eth {
     const params = { timestamp: timestampRangeParams };
 
     // get contract results logs using block timestamp range
-    const logs = await this.common.getLogsWithParams(null, params, requestIdPrefix);
+    const logs = await this.getLogsWithParams(null, params, requestIdPrefix);
```

Comment:
Why this changes?? seems like they are not from your changes 😅

Reply:
This change was made by Konstantina three weeks ago.

<img width="719" alt="Screenshot 2023-08-16 at 2 32 58 PM" src="https://github.com/hashgraph/hedera-json-rpc-relay/assets/183383/2959093d-27ac-48d2-9b01-b39fe7a23080">